services/frontend-v3/src/i18n/fr_CA.json

```
@@ -160,6 +161,7 @@
   "landing.description": "I-Talent est une plateforme pour mettre en valeur votre talent et vos compétences au sein de l'ISDE! C'est un moyen sécuritaire et facile de créer et de maintenir votre profil. ",
   "landing.call.to.action": "Connectez-vous avec d'autres personnes talentueuses pour apprendre et grandir ensemble.",
 
+  "select.any": "Toute",
```

In French, the masculine takes over the feminine form for general things
```suggestion
  "select.any": "Tout",
```